kafka-rest/src/main/java/io/confluent/kafkarest/controllers/ControllersModule.java

```diff
@@ -27,7 +27,7 @@ protected void configure() {
     bind(ClusterManagerImpl.class).to(ClusterManager.class);
     bind(PartitionManagerImpl.class).to(PartitionManager.class);
     bind(ReplicaManagerImpl.class).to(ReplicaManager.class);
-    bind(TopicConfigurationManagerImpl.class).to(TopicConfigurationManager.class);
+    bind(TopicConfigManagerImpl.class).to(TopicConfigManager.class);
```

Comment:
I think the name of this class should remain unchanged: `TopicConfigurationManagerImpl`.

Reply:
We changed the name of the referred abstraction to Config (plural Configs). We should not refer to the same thing by different names. That would be highly confusing. Acknowledged.